module/VuFind/src/VuFind/Db/Table/AuthHash.php

```diff
@@ -108,14 +108,14 @@ public function getLatestBySessionId($sessionId)
     /**
      * Update the select statement to find records to delete.
      *
-     * @param Select $select  Select clause
-     * @param int    $daysOld Age in days of an "expired" record.
+     * @param Select $select    Select clause
+     * @param string $dateLimit Date threshold of an "expired" record in format
+     * 'Y-m-d H:i:s'.
      *
      * @return void
      */
-    protected function expirationCallback($select, $daysOld)
+    protected function expirationCallback($select, $dateLimit)
     {
-        $expireDate = time() - $daysOld * 24 * 60 * 60;
-        $select->where->lessThan('created', date('Y-m-d H:i:s', $expireDate));
+        $select->where->lessThan('created', strtotime($dateLimit));
```

Comment:
Why are you calling strtotime here? It seems like you probably shouldn't...

Reply:
Oops, fixed. Also tested that it works correctly now. Previously it looked like it worked but didn't remove anything. Apparently a bad value like that will get truncated with just a warning that's not visible anywhere.